deployment/docker-compose/docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
       - .core_backend.env
     environment:
       - REDIS_HOST=redis://redis:6379
+      - LITELLM_ENDPOINT=http://huggingface-embeddings
```

Comment:
I don't think we should bake that in otherwise it will override the one in .core_backend.env. But they will probably have their own server and they won't user docker-compose. I will update it.

Reply:
This is so in .core_backend.env we keep values for local setup (make setup-dev), and we can override it for docker-compose only in docker-compose.yml.